debian/rules

```diff
@@ -8,6 +8,10 @@ export PYBUILD_NAME=eos-data-distribution
 
 override_dh_install:
 	dh_install --fail-missing
+	mkdir -p debian/edd-avahi-monitor/usr/bin
```

Comment:
I believe you could also set `PYBUILD_DESTDIR = debian/tmp`. In either case, you'd need to explicitly add the python files to the .install file.